README.md

```diff
@@ -8,14 +8,22 @@
 
 # 1
 To start the project run:
-`bash create-vm.sh`
+
+```shell
+bash create-vm.sh
+```
 
 # 2
 To transfer file from local to vm, cd to Project-AWS and run the following command:
-`multipass transfer TFInstall.sh awesome-vm:/home/ubuntu`
+
+```shell
+multipass transfer terraformInstall.sh awesome-vm:/home/ubuntu
+```
 
 # 3
 To install Terraform in the VM shell into your multipass VM and run the TFInstall.sh script: 
-`multipass shell awesome-vm`
-`bash TFInstall.sh`
 
+```shell
+multipass shell awesome-vm
+bash terraformInstall.sh
```

Comment:
You could do this as a `multipass exec...`

Reply:
Will address in a separate branch